src/Contracts/AuthorizationInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
I am not sure these interfaces are useful.
I may be wrong, but no one is supposed to create its own Authorization or Subscription class

Reply:
So my thoughts for this are, there are two classes that are internal to this library which are interacted with outside of `WebPush`; being `Subscription` and `Authorization`. 

Each of which, given the example use cases in the README, are being fetched from some kind of persistence layer or storage of some sort and then the data is parsed and used to build up each of these objects. 

By providing these interfaces, anyone who wishes to take advantage can implement them on their data models and then simply pass those data models in place of `Authorization` and `Subscription` and not have to do any parsing or manipulation.